Source/Core/DolphinQt/TAS/GCTASInputWindow.cpp

```diff
@@ -30,17 +33,7 @@ GCTASInputWindow::GCTASInputWindow(QWidget* parent, int num) : TASInputWindow(pa
   top_layout->addWidget(m_main_stick_box);
   top_layout->addWidget(m_c_stick_box);
 
-  m_triggers_box = new QGroupBox(tr("Triggers"));
-
-  auto* l_trigger_layout =
-      CreateSliderValuePairLayout(tr("Left"), m_l_trigger_value, 0, 255, Qt::Key_N, m_triggers_box);
```

Comment:
CreateSliderValuePairLayout() sets the supplied QSpinBox* (m_l_trigger_value here) to the address of the QSpinBox created by CreateSliderValuePair(), but you don't do that below leaving it as a null pointer. This causes a crash when a game is running and TAS input is active, as TASInputWindow::GetSpinBoxU16 tries to dereference a null pointer derived from that QSpinBox.

Ditto for the other removed calls to CreateSliderValuePairLayout.